pkg/logs/auditor/auditor.go

```diff
@@ -31,13 +31,15 @@ const registryAPIVersion = 2
 // Registry holds a list of offsets.
 type Registry interface {
 	GetOffset(identifier string) string
+	GetTailingMode(identifier string) string
 }
 
 // A RegistryEntry represents an entry in the registry where we keep track
 // of current offsets
 type RegistryEntry struct {
 	LastUpdated time.Time
 	Offset      string
+	TailingMode string
```

Comment:
Because this is JSON and because this is a new field, there won't be any issue as long as we properly handle the default `""` value, however, I'm wondering if we should bump the `Version` in the `registry.json` file? I don't know the semantic of this version field, maybe @ajacquemot could give us some info on this?